### DISCUSSION TITLE
Google Ad Manager and iOS

### DISCUSSION THREAD

**Discussion by tienbrown** [marcojak/MauiMTAdmob#106] [Q&A]
Has anyone had success using this plugin for Google Ad Manager instead of Admob, specifically iOS? I have been able to successfully use the Xamarin version of this plugin for one of my applications and it currently returns ads on iOS. I have since converted the application to Maui and am using this plugin and unfortunately not able to get an add to return on iOS. The add returns fine on Android. Test ads run display fine on iOS. I receive the following on ad failed to load:

txparks.Utility.Logging.LoggingService: Debug: [TabList] ..OnAdsFailedToLoad
txparks.Utility.Logging.LoggingService: Error: [TabList] 1:Error Domain=com.google.admob Code=1 "Request Error: No ad to show." UserInfo={NSLocalizedDescription=Request Error: No ad to show., gad_response_info=  ** Response Info **
    Response ID: (null)
    Network: (null)

  ** Mediation line items **
}

Thanks for any suggestions/responses.

**Comment by marcojak** [comment]
Hi.
If the test ads work fine, then it's all working correctly. It might be possible that you are not getting real ads because the app is not on the store yet or simply there were no ads to show in that moment.

I'm planning to add full support to Ad Manager in the next version together with .Net 9.
